css/index.css

```diff
@@ -0,0 +1,14 @@
+.row
```

Comment:
When defining a CSS ruleset, you need to surround the whole block  in curly braces {}. Here, it looks like you're styling a class called row, but the ruleset isn't surrounded in braces.